pkg/compare/testdata/UserOverride/localout.golden

```diff
@@ -0,0 +1,46 @@
+**********************************
+
+Cluster CR: v1_Namespace_openshift-something-else
+Reference File: namespace-no-patch.yaml
+Diff Output: diff -u -N TEMP/v1_namespace_openshift-something-else TEMP/v1_namespace_openshift-something-else
+--- TEMP/v1_namespace_openshift-something-else	DATE
++++ TEMP/v1_namespace_openshift-something-else	DATE
+@@ -2,8 +2,20 @@
+ kind: Namespace
+ metadata:
+   annotations:
+-    somethingelse: true
+-    workload.openshift.io/allowed: management
++    openshift.io/sa.scc.mcs: s0:c29,c14
++    openshift.io/sa.scc.supplemental-groups: 1000840000/10000
++    openshift.io/sa.scc.uid-range: 1000840000/10000
++    reclaimspace.csiaddons.openshift.io/schedule: '@weekly'
+   labels:
+-    openshift.io/cluster-monitoring: "true"
++    kubernetes.io/metadata.name: openshift-storage
++    olm.operatorgroup.uid/ffcf3f2d-3e37-4772-97bc-983cdfce128b: ""
++    openshift.io/cluster-monitoring: "false"
++    pod-security.kubernetes.io/audit: privileged
++    pod-security.kubernetes.io/audit-version: v1.24
++    pod-security.kubernetes.io/warn: privileged
++    pod-security.kubernetes.io/warn-version: v1.24
++    security.openshift.io/scc.podSecurityLabelSync: "true"
+   name: openshift-something-else
++spec:
++  finalizers:
++  - kubernetes
+
+**********************************
+
+Cluster CR: v1_Namespace_openshift-storage
+Reference File: namespace.yaml
+Diff Output: None
+Patched with testdata/UserOverride/localnewOverridesout.golden
+Patch Reasons: <None given>
```

Comment:
Should we maybe fail in case of "None given" to force some documentation of the patches?

Reply:
Do you mean fail to generate them or fail to report on a diff?

Reply:
We could refuse to add the patch if no reason is given and then put an error/warning log to explain

Reply:
I still think forcing "some string" is better than allowing nothing.  We can't stop people from lying about the reason, but we can at least encourage some kind of good behavior.